utility/ln_config.h

```diff
@@ -55,6 +55,21 @@
  // figure out what board we are building
 
  // Common defines
+ 
+// turn off EEPROM for platforms not currenty implemented
+#if defined(STM32F1) || !defined(ES8266) || !defined(__AVR__) 
+  #ifndef LOCONET_NO_EEPROM 
+  #define LOCONET_NO_EEPROM
+  #endif
+#endif
+
+// turn off Software UART for platforms not currenty implemented
```

Comment:
This also seems wrong to #define LOCONET_NO_SW_UART by default when there clearly is a Bit-Bashed Software UART for these platforms. I don't understand why you want to do this

Reply:
same as above, as software UART is only implemented for STM32F1, ESP8266 and AVR.

Why I want to do this: e.g. when using the library on an ESP32 or Raspberry PICO you may implement the transport channel on your own. 
I'm using it on an ESP32 with LOCONET via a bluetooth serial port (works great) and also via TCP/IP socket (virtual serial port on PC forwarding communication to a socket).